src/app/components/Rule.tsx

```diff
@@ -25,7 +25,7 @@ import DebugLogger from "@/app/components/DebugLogger";
 import useGlobalState from "@/app/hooks/useGlobalState";
 import { isDark, Theme } from "@/app";
 
-type RuleType = "force" | "rollout" | "experiment" | "prerequisite";
+type RuleType = "force" | "rollout" | "experiment" | "prerequisite" | "Safe Rollout";
```

Comment:
why is this the only value with a different style of writing? i.e. why not use `safe-rollout` as it is the RuleType we have in the back-end code too.